### PR TITLE
New feature: Copy all to clipboard

### PR DESCRIPTION
**Pull request by irsl**
## Description
Putty features "Copy all to clipboard" to put all lines in the current session to the clipboard. It is now exposed via mRemoteNg as well.

## Motivation and Context
This is a new feature in mRemoteNg.
https://github.com/mRemoteNG/mRemoteNG/issues/1209

## How Has This Been Tested?

The patch is light-weight. Only manual testing was done

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: mRemoteV1/UI/Window/ConnectionWindow.cs [hunk 1]
```diff
@@ -419,11 +421,11 @@ private void ShowHideMenuButtons()
 
                 if (interfaceControl.Protocol is PuttyBase)
                 {
-                    cmenTabPuttySettings.Visible = true;
+                    cmenCopyAllToClipboard.Visible = cmenTabPuttySettings.Visible = true;
```
Comment (kmscode) [inline mRemoteV1/UI/Window/ConnectionWindow.cs]: These options should be in addition to cmenTabPuttySettings.Visible not in place of them...

**Comment by kmscode** [comment]
Thanks for the PR!!!!

I did a super quick code review and made one comment in the code that needs to be fixed. If you can't get to that let me know and I'll take care of it.

Once that's done I'll review and test further.

Thanks again!

File: mRemoteV1/UI/Window/ConnectionWindow.cs (hunk 1 above)
Comment (irsl) [reply mRemoteV1/UI/Window/ConnectionWindow.cs]: Sorry, I don't really get it. Would you prefer this dissected in two lines?

Comment (kmscode) [reply mRemoteV1/UI/Window/ConnectionWindow.cs]: I did say "a super quick code review" in my other comment... :wink:

I missed the inline setter calls for both components. All good here...

**Comment by kmscode** [comment]
@irsl, we do not commit directly to mater. Reference: https://github.com/mRemoteNG/mRemoteNG/wiki/Git-Workflow

Could you please resubmit to the develop branch and I will gladly approve the PR. Thanks again!!!

**Comment by farosch** [comment]
Closed in favor of #1397

**Pull request closed without merging**